spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskJobService.java

```diff
@@ -106,10 +107,18 @@ public DefaultTaskJobService(JobService jobService, TaskExplorer taskExplorer,
 	}
 
 	@Override
-	public List<TaskJobExecution> listJobExecutionsWithStepCount(Pageable pageable) {
+	public List<TaskJobExecution> listJobExecutionsWithStepCount(String queryString, Pageable pageable) {
```

Comment:
We'll need a unit test for this method as well.

Reply:
Can you help me out with where the test cases have been added for other methods in this class? The DefaultTaskJobServiceTests class covers only the constructor in DefaultTaskJobService.